docs/Scaling/overview.md

```diff
@@ -0,0 +1,54 @@
+--8<-- "References/abbreviations.md"
+
+# Scaling Overview
+Scaling functionality in MPS enables Open AMT Cloud Toolkit to support a greater number of managed devices. The toolkit offers various methods for deploying scaling, including Kubernetes (K8s), Azure Kubernetes Service* (AKS), and Docker Swarm*. In addition, administrators can use kuberctl to manage the AKS. 
+
+
+![../assets/images/HighLevelArchitectureScaling](../assets/images/HighLevelArchitectureScaling.png)
+
+**Figure 1: High-level Architecture of Scaling Implementation**
+
+Figure 1 illustrates the basic high-level software flow:
+
+* Managed devices use CIRA to connect and call home to instances of the MPS in the cloud. 
+* RPCs connect to an available instance of the MPS Server with WSS calls. These calls are funneled through Kong* API Gateway, which supports a variety of APIs. Kong manages load balancing, logging, authentication and more. 
+* The Kong* API Gateway handles requests from client apps, such as the Sample Web UI included in Open AMT Cloud Toolkit, sending them along to an available RPS.
+* The MPS Router chooses an available instance of the MPS.
+* The RPS microservices communicate with MPS microservices through the REST API. 
```

Comment:
When a managed device is activated, RPS registers the device information with MPS using the MPS REST API.